### PR TITLE
Add Reorder to Activity Collection

### PR DESCRIPTION
**Pull request by m6jones**

### CODE REVIEW

**Comment by AKobets** [comment]
Is anything outside of learning path scope is affected by this PR ?

File: components/d2l-activity-collection-editor/d2l-activity-collection-editor.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { ifDefined } from 'lit-html/directives/if-defined';
 import { repeat } from 'lit-html/directives/repeat';
 import { until } from 'lit-html/directives/until.js';
 import { guard } from 'lit-html/directives/guard';
+import { classMap } from 'lit-html/directives/class-map.js';
```
Comment (zinabat) [inline components/d2l-activity-collection-editor/d2l-activity-collection-editor.js:6]: Nit: imports should be alphabetical

Comment (m6jones) [reply components/d2l-activity-collection-editor/d2l-activity-collection-editor.js:6]: There is no linter here? :( Will fix

**Comment by m6jones** [comment]
This is learning paths only

**Pull request merged**